persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -85,55 +79,20 @@ public void execute() {
             return;
         }
 
-        if (datastore == null) {
-            datastore = PersistToolsConstants.SupportedDataSources.IN_MEMORY_TABLE;
-        } else if (!SUPPORTED_DB_PROVIDERS.contains(datastore)) {
-            errStream.printf("ERROR: the persist layer supports one of data stores: %s" +
-                    ". but found '%s' datasource.%n", Arrays.toString(SUPPORTED_DB_PROVIDERS.toArray()), datastore);
-            return;
-        }
-
         if (Files.isDirectory(Paths.get(sourcePath, PERSIST_DIRECTORY, MIGRATIONS))) {
             errStream.println("ERROR: reinitializing persistence after executing the migrate command is not " +
                     "permitted. please remove the migrations directory within the persist directory and try " +
                     "executing the command again.");
             return;
         }
```

Comment:
Shall we remove this too? As we are not adding any configuration, this validation is not valid

Reply:
Fixed in [5a7297f](https://github.com/ballerina-platform/persist-tools/pull/336/commits/5a7297f297e0a3e36d42d91d70bb4bbf78856fb1)